script/e2e-test.sh

```diff
@@ -176,9 +191,9 @@ installOrUpgradeKubeapps() {
     "${img_flags[@]}"
     "${@:2}"
     "${multiclusterFlags[@]+"${multiclusterFlags[@]}"}"
-    --set frontend.replicaCount=2
-    --set kubeops.replicaCount=2
-    --set dashboard.replicaCount=2
+    --set frontend.replicaCount=1
+    --set kubeops.replicaCount=1
+    --set dashboard.replicaCount=1
```

Comment:
Hmm, it might be worth checking the commit msg there? I remember some environment we found during release was returning 502s, but it may have been for the kubeappsapis service only, which you've left with 2 below.

Reply:
Previously, only service with 2 replicas was kubeappsapis indeed.